src/Service.Host/client/src/components/StorageLocation.js

```diff
@@ -124,6 +127,17 @@ function StorageLocation({ creating }) {
         setChangesMade(true);
     };
 
+    const okToSave = () => {
+        if (creating) {
+            return formValues.siteCode && formValues.storageAreaCode && formValues.accountingCompany && formValues.description;
+        }
+        return true;
+    }
+
+    const makeInvalid = () => {
+        handleFieldChange("dateInvalid", new Date());
```

Comment:
is your eslint working? these should be single quotes for example

Reply:
Eslint is installed and was working but i don't get the red squiggles under these lines

Reply:
do you have the prettier extension?